### DISCUSSION TITLE
[India] ascii n hscii font support.

### DISCUSSION THREAD

**Discussion by zawa8** [gsantner/markor#1522] [Ideas]
vhen ve do settings edit mode -> font .
there font can be decimal no system + vhich phonetic language.
so user should check or   hexfont/decimalfont (hscii8210/ascii5210 )

[video for k88 n markor](https://youtu.be/RG3lLzC6fhA).

and if font style is provided for any selected text , then zust by font multilingual can be achieved in a single file for Aal languages vhich are phonetic based.
also kindly convert your app to vhatsapp like messaging app. please....

rigards,
viml

![Screenshot_2021-11-28-17-23-45-059_net gsantner markor](https://user-images.githubusercontent.com/81591228/144726642-8d30b383-73da-4956-967b-2b194659bbfe.png)
![Screenshot_2021-11-28-17-24-29-646_net gsantner markor](https://user-images.githubusercontent.com/81591228/144726644-ed01d928-6cb5-4287-b6a1-322b27d88a45.png)
).

**Comment by gsantner** [comment]
Hello!

Sorry to say but I don't understand any asian languages/words/characters, and I don't really get it what you want or your issue is.  
But looking at the video - I can say that I can't help you. If you can work it out on your own and make a pull request I'm happy to review it and merge it if it fits.

Thanks
